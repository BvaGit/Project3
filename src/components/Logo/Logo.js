import React from 'react';
import './Logo.scss';

const Logo = (props) => {
    const { src, alt} = props;
        return (
            <img src={src} alt={alt} className="logo"/>
        )
}

export default Logo;
