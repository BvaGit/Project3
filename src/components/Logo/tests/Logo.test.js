import React from 'react';
import { shallow, mount } from 'enzyme';
import Logo from '../Logo';


describe('Logo', () => {
    it("Should match snapshot", () => {
        const component = shallow(<Logo />)
        expect(component).toMatchSnapshot()
    })
})