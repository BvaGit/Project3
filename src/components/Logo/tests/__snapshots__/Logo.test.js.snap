// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo Should match snapshot 1`] = `ShallowWrapper {}`;
